src/spikeinterface/sorters/basesorter.py

```diff
@@ -271,7 +268,8 @@ def run_from_folder(cls, output_folder, raise_error, verbose):
         if has_error and raise_error:
             print(log["error_trace"])
             raise SpikeSortingError(
-                f"Spike sorting failed. You can inspect the runtime trace in {output_folder}/spikeinterface_log.json"
+                f"Spike sorting failed. You can inspect the runtime trace in {output_folder}/spikeinterface_log.json. "
+                f"Error trace: {log['error_trace']}"
```

Comment:
Not sure this is a good idea because this error_trace can be enourmous and not well formated.
This is why the error was short and we used a print(log["error_trace"]) on line before

Reply:
Ah, but it's not clear that that's the error! I'll remove the previous print and keep it like this. Usually the error trace is not too long, and if it is it' good to print it!

Reply:
done